dev/DropDownButton/DropDownButton.xaml

```diff
@@ -62,6 +62,9 @@
                                         <ObjectAnimationUsingKeyFrames Storyboard.TargetName="ContentPresenter" Storyboard.TargetProperty="Foreground">
                                             <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource ButtonForegroundPressed}" />
                                         </ObjectAnimationUsingKeyFrames>
+                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="ChevronTextBlock" Storyboard.TargetProperty="Foreground">
+                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource TextFillColorTertiaryBrush}" />
```

Comment:
We shouldn't be using the new theme colors directly -- should this just be ButtonForegroundPressed?

Reply:
The chevron colour seems to be a shade lighter, so is the tertiary brush colour when pressed. I can create a new themeresources for DropDownButton, or reference the `SplitButtonForegroundSecondaryPressed` brush that I created?

Reply:
Oh, weird. Yeah, create a new resource for DropDownButton then. :)